primitives/io/src/lib.rs

```diff
@@ -408,6 +408,105 @@ pub trait Misc {
 	}
 }
 
+/// for times you want your chain to be useless
+#[runtime_interface]
+pub trait UselessCrypto {
```

Comment:
Ideally this should not live in Substrate.

If it needs to live in Substrate, it should go into a different crate. Where it is directly understandable that this is for testing.